docs/product/upgrade-guide.md

```diff
@@ -0,0 +1,75 @@
+<!--
+title: Upgrade Guide
+menuText: Upgrade Guide
+description: Upgrading Serverless Console for October release
+menuOrder: 7
+-->
+# Upgrade Guide
+
+On **Tuesday, October 25th at 10AM PDT**, we will be releasing a new version
+of Serverless Console which will break existing deployments with the Serverless
+Console extension. There are a few things you will need to do before and after
+the release of the new Serverless Console.
+
+## Action items before October 25th at 10AM PDT:
+- You MUST remove `console: true` from all `serverless.yml` files. Deployments
+after October 25th at 10AM will fail if `console: true` is present in the
+`serverless.yml`. You can continue to deploy with `console: true` until then.
+You can also continue to use Serverless Console without change until then.
+
+- If you don’t also use Serverless Dashboard, then you SHOULD remove
+`org: <org>` from all `severless.yml` files.
+- Join the [Console community channel in Slack](https://serverless-contrib.slack.com/archives/C037D989FB5)
+for up to date changes. We are doing our best to release at this time, but if
+there are changes you’ll be able to get the latest in Slack.
+
+## Action items after October 25th at 10AM PDT:
+- You MUST upgrade Serverless Framework to version 3.24.0 or greater with
+`npm i serverless --global`.
+- You MUST run `serverless --console`, or visit console.serverless.com, to
+reconnect your AWS account after October 25th at 10AM.
+
+## FAQ
+
+### Why is this a breaking change?
```

Comment:
I'd recommend we add a bit of spin here. e.g. 

"We are changing how Serverless Console integrates with AWS to improve performance and broaden the scope of observable resources. This means..."